.github/workflows/publish-major-minor.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from packaging.version import parse
+
+print(sys.version_info)
+tag_ref = sys.argv[1]
+tag_name = tag_ref.split("/")[-1]
+print(f"tag_name: {tag_name}")
+version = parse(tag_name)
+print(f"version: {version}")
+if not (version.is_prerelease):
+    print("Creating new major and minor tags!")
+    print(f"::set-output name=original_tag_name::{tag_name}")
+    print(f"::set-output name=major_version::v{version.major}")
+    print(f"::set-output name=minor_version::v{version.major}.{version.minor}")
```

Comment:
Maybe this should be grouped? as in https://github.com/deadsnakes/action/blob/dafca65/bin/install-python#L21-L27

Reply:
As I understand it, the grouping is used to hide output in these little dropdown groups.
But `::set-output` (even so it is printed) doesn't generate output in the log.
In total  there are 3 lines in the log (besides the script itself), e.g.:
```
tag_name: v2.0.0
version: 2.0.0
Creating new major and minor tags!
```
Which IMHO should be fine (thinking back on travis or appveyor logs :dizzy_face:)

Reply:
Fair enough